core/domain/collection_domain.py

```diff
@@ -1112,6 +1112,25 @@ def get_node(self, exploration_id):
                 return node
         return None
 
+    def get_sorted_nodes(self):
```

Comment:
Maybe instead of ``get_sorted_nodes``, what about ``get_nodes_in_playable_order``? It may not be clear in the current name what the sort criteria is.